config/redis.yml

```diff
@@ -65,6 +65,9 @@ development: &defaults
   user_account_details:
     namespace: user-account-details
     each_ttl: 1296000 # 15 days
+  evss_dependents_retrieve_response:
+    namespace: evss-dependents-retrieve-response
+    each_ttl: 1800
```

Comment:
What should the proper `ttl` be?

Reply:
i'm going to ask about this in the 686 channel, depends on if users can retrieve their information immediately after submitting

Reply:
how about making the ttl 1 day and deleting the cache after successful submission?